src/main/scala/diplomacy/CloneModule.scala

```diff
@@ -7,6 +7,7 @@ package chisel3.shim
 
 import Chisel._
 import chisel3.experimental.BaseModule
+import chisel3.reflect.DataMirror
```

Comment:
This is the Chisel 3.6 import, it's `chisel3.experimental.DataMirror` in Chisel 3.5 and before.